### ISSUE TITLE
`no-unused-variable` with JSX dot syntax

### ISSUE THREAD

**Issue by dallonf** [palantir/tslint#1010]
This example will flag the `FormControls` import with `no-unused-variable`:

```ts
import React from 'react';
import {FormControls} from 'react-bootstrap';

export default class Example extends React.Component<{}, {}> {
  render() {
    return <FormControls.Static label="Hello" value="World" />;
  }
}
```

This is a pattern used in a few UI libraries, like React Bootstrap as shown in the example.

**Comment by JKillian** [comment]
There is most likely a bug in the TypeScript language services that causes this bug in TSLint. I'll see if I can make a good minimal repro and submit a bug with TypeScript if there's not one already

**Comment by myitcv** [comment]
@dallonf - what version of TypeScript are you using here?

**Comment by dallonf** [comment]
It's 1.8.7 in the project... but now that you mention it, the warning only seems to appear in Atom, using the [linter-tslint](https://atom.io/packages/linter-tslint) and [atom-typescript](https://atom.io/packages/atom-typescript) packages.

(Running `tslint` from a npm script, which uses versions specified in `package.json`, does not show the warning)

I don't know what version those use, but they seem to track latest. So your hypothesis about this being a TypeScript bug is probably correct.

**Comment by JKillian** [comment]
I just tested with the latest nightly, `typescript@1.9.0-dev.20160322`, and unfortunately I'm still seeing this bug. I'll file an issue with them if there's not one around already

**Comment by mpseidel** [comment]
+1 using typescript@^1.9.0-dev.20160129


`import * as shared from "./shared/index";`

`const Footer = props => <shared.ItemFooter {...props} />;`

=> unused variable: 'shared'

**Comment by adidahiya** [comment]
@mpseidel that is a pretty old nightly. [This comment](https://github.com/Microsoft/TypeScript/issues/7637#issuecomment-200469241) suggests it was fixed in the compiler in late March. Can you give that a try to verify the fix?

**Comment by mpseidel** [comment]
@adidahiya sorry - updated to the latest build and now it works like a charm. thanks for the heads-up! :)

**Comment by adidahiya** [comment]
sweet, closing this issue out
